src/parsers/axis.js

```diff
@@ -49,6 +50,7 @@ export default function(spec, scope) {
     scale:  scope.scaleRef(spec.scale),
     extra:  config.tickExtra,
     count:  scope.objectProperty(spec.tickCount),
+    integer: scope.property(spec.integer),
```

Comment:
This line does not reference the configuration property. However, I think that is fine, as I don't think we need or want a configuration entry for this.